src/v2/stores/socket.js

```diff
@@ -0,0 +1,22 @@
+import {initSocket} from 'v2/api/socket';
+import networkOverview from 'v2/stores/networkOverview';
+import nodes from 'v2/stores/nodes';
+
+const socketActions = {
```

Comment:
this should be refactored a bit because the socket isn't a different store; each message can be retrieved in one of 2 ways: (a) via API for initial population or (b) incoming real-time via the socket. I think the model ends up being similar to initial population of chat history in a chat app